webapp/channels/src/actions/cloud.tsx

```diff
@@ -48,7 +48,7 @@ export function completeStripeAddPaymentMethod(
                             line2: billingDetails.address2,
                             city: billingDetails.city,
                             state: billingDetails.state,
-                            country: getCode(billingDetails.country),
+                            country: billingDetails.country,
```

Comment:
The country field is already storing it as the ISO code, so there was no need for this getCode call.